src/averageAges.js

```diff
@@ -15,13 +15,21 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  let mens = people.filter(person => person.sex === 'm');
+
+  if (century !== undefined) {
```

Comment:
The division by zero is already handled with `parseFloat((totalAge / mens.length).toFixed(2));` because it returns NaN if `mens.length` is 0, so you can remove the explicit check for division by zero.